slides/en/index.html

```diff
@@ -281,11 +318,14 @@ <h2>Gateways</h2>
               <img class="no-margin" src="images/bpmn_gateway_design.png" alt="Symetrical gateway design"/>
 
               <aside class="notes">
-                - Exclusive: use default TR and make sure only one condition is true among outgoing TRs<br/>
-                - Exclusive: only one incoming TR should reach the gateway.<br/>
-                - Parallel: all outgoing TR fired at the same time (no condition allowed).<br/>
-                - Parallel: all incoming TR are waited for even if some can never be reached due to bad design.<br/>
-                - Inclusive: used when dealing with parallel gateways with conditions
+		        5<br /><br />
+                Exclusive: use default TR and make sure only one condition is true among outgoing TRs<br/>
+                Exclusive: only one incoming TR should reach the gateway.<br/>
+                Parallel: all outgoing TR fired at the same time (no condition allowed).<br/>
+                Parallel: all incoming TR are waited for even if some can never be reached due to bad design.<br/>
+                Inclusive: used when dealing with parallel gateways with conditions<br />
+				Best practice: explicit, symmetric and default<br />
```

Comment:
Can you use 2 spaces for indentation rather then tab?

Reply:
Good point